resources/js/components/app-form-modal.js

```diff
@@ -123,7 +125,7 @@ class AppFormModal extends LitElement {
         }
 
         .show-modal {
-            background-color: unset;
+            background-color: var(--modal-background-color);
             opacity: 1;
```

Comment:
@Perumal605, let's add a bit of transparency so we can see the background behind the screen.